src/sagas/wallet.js

```diff
@@ -537,6 +547,8 @@ export function* setupWalletListeners(wallet) {
     }));
 
     return () => {
+      // XXX: Is this cleanup working?
```

Comment:
It seems like it's not working. Should we create an issue/KTLO for that?

Reply:
It is a memory leak, if the handlers are being kept.
Although I don't think it is a "serious" memory leak since it only happens when we are starting the wallet, which should not happen many times during the app lifecycle.

I'll create an issue for this and link to this line, what do you think?